textinput.go

```diff
@@ -31,6 +30,10 @@ type textinputPlugin struct {
 	selectionExtent int
 }
 
+type keyboardShortcutsGLFW struct {
```

Comment:
It's not directly clear to me what this type is for and how it works (implemented in multiple environments). Perhaps a comment explaining that would be nice for future readers of this code.